### PR TITLE
align sorting of success level

### PR DESCRIPTION
**Pull request by AsebWebDev**
I aligned the new success level to have the lowest level. I think all other levels should come after that, because they increase in relevance for debugging :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,21 +109,24 @@ logger.colorLog({
...
 // debug level, with prefix "[DEBUG]"
 logger.debug(message);
-// Error level, with prefix "[ERROR]"
-logger.error(message);
 // Info level, with prefix "[INFO]"
 logger.info(message);
 // Warn level, with prefix "[WARN]"
 logger.warn(message);
+// Error level, with prefix "[ERROR]"
+logger.error(message);
```
Comment (AsebWebDev) [inline README.md:128]: setting `error` as highest escalating level

**Comment by tigercosmos** [comment]
ummm... I misunderstood the purpose of  `success()` 
could you also help me review the README and revise the relevant content? (like https://github.com/tigercosmos/node-color-log/commit/4a0c084ebe850e50f3a00bb570628ee714461649)

**Comment by AsebWebDev** [comment]
I might have also misunderstood the usage of levels, so I might be wrong. But as far as I understood setting the level f.e. with `.setLevel('info')` will only show logs above equal or above that level. 

And my understanding of `success` would be, that it is used for a log, that indicates a successful operation. If it is not a successful operation, we might want to debug or show an info. Next escalation level would be warning and finally an error. 

That is why I have been suggesting to have this ascending order. :) 

So using `.setLevel('success')` would show all logs. Using `.setLevel('info')` will hide all `success` and `debug` logs. Am I right :)

**Comment by tigercosmos** [comment]
ok. I understood.

**Comment by tigercosmos** [comment]
9.0.0

**Pull request merged**